common/js/src/i18n.js

```diff
@@ -41,6 +41,9 @@ var I18N_DATA_ATTRIBUTE = 'data-i18n';
 /** @const */
 var I18N_DATA_ARIA_LABEL_ATTRIBUTE = 'data-i18n-aria-label';
 
+/** @const */
+var I18N_TITLE_ATTRIBUTE = 'title';
```

Comment:
I think we want this to be "data-title", so that there's no confusion between translated and untranslated text.

If we use the same attribute "title" as it is now, we'll have a problem that calling adjustElementTranslation() twice would call something like `chrome.i18n.getMessage('Initializing...')` and fail.